app/src/main/java/org/dieschnittstelle/mobile/android/skeleton/model/LocalTaskDatabaseOperation.java

```diff
@@ -78,6 +79,9 @@ public boolean updateTask(Task task) {
 
     @Override
     public boolean deleteTask(long id) {
-        return false;
+        Task task = new Task();
+        task.setId(id);
+        taskDatabase.getDao().deleteTask(task);
```

Comment:
wird hier das Task mit dem ID von Datenbank gelöscht? weil, es sieht so aus dass ein neues Task mit demselben ID erstellt und gelöscht wird und dem richtigen Task mit dem ID immer noch in DB gespeichert ist